nixos-modules/microvm/options.nix

```diff
@@ -483,6 +483,13 @@ in
       '';
     };
 
+    vmHostPackages = mkOption {
+      description = "If set, overrides the default host package.";
```

Comment:
Excuse me for not knowing how a Mac works, but how exactly is this going to be used? Is that really enough documentation to be obvious all nix-darwin users? :-)

Reply:
Yes the docs are quite lacking... I'm going to need to do a more detailed writeup